root/static/css/form.css

```diff
@@ -1126,3 +1127,46 @@ form li.complex p.error{
 	font-weight:normal;
 	font-size:160%;
 }
+
+/*responsive*/
+@media all and (min-width: 50px) and (max-width: 400px){
+  form ul li.rightHalf {
+    width: 97% !important;
+  }
+}
+/*@media all and (min-width: 300px) and (max-width: 650px) {
+  table .responsive { 
+    margin: 5px 10px;
+    border: 1px solid #aaa;
+    border-width: 1px 1px 0 0;
+    border-spacing: 0;
+    width: 200px;
+  } 
+}
+@media all and (min-width: 90px) and (max-width: 299px) {
+  table .responsive { 
+    margin: 5px 10px;
+    border: 1px solid #aaa;
+    border-width: 1px 1px 0 0;
+    border-spacing: 0;
+    width: 100px;
+  } 
+}*/
```

Comment:
Comments should help better explain the logic/code. What is the purpose of this commented code?